src/proto/single/mod.rs

```diff
@@ -182,6 +183,24 @@ impl JobBuilder {
         self.try_build()
             .expect("All required fields have been set.")
     }
+
+    /// Builds a new _trackable_ `Job``.
+    ///
+    /// Progress update can be sent and received only for the jobs that have
+    /// been explicitly marked as trackable via `"track":1` in the job's
+    /// custom hash.
+    /// ```
+    /// use faktory::JobBuilder;
+    ///
+    /// let _job = JobBuilder::new("order")
+    ///     .args(vec!["ISBN-13:9781718501850"])
+    ///     .build_trackable();
+    /// ```
+    #[cfg(feature = "ent")]
+    pub fn build_trackable(&mut self) -> Job {
+        self.add_to_custom_data("track", 1);
+        self.build()
+    }
```

Comment:
What do you think of instead just having
```rust
if cfg!(feature = "ent") {
    self.add_to_custom_data("track", 1);
}
```
inside the regular `build` function? Is there ever a reason to _not_ mark a job as trackable?

If not, I would probably not make this a separate build method, and instead just have it be a builder-style method like:
```rust
impl JobBuilder {
    pub fn trackable(self, yes: bool) -> Self
}
```

Reply:
Not that I can come up with, but people _may_ want to have some of the jobs untrackable.. As for the terminal method, I just wanted to save them a method call: instead of `another_method().trackable().build()` simply `another_method().build_trackable()`.

After thinking it over a little bit, I realized that those who have paid for the Ent Faktory subscription will want all the paid features included by default and should not opt in additionally. I got rid of the `build_trackable` and  updated the logic and tests accordingly. Also updated the docs for `build` mentioning how they can opt out of tracking.